sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -131,6 +136,26 @@
             .publishOn(scheduler);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Mono<ServiceBusReceivedMessage> receiveDeferredMessage(ReceiveMode receiveMode, long sequenceNumber) {
+        return receiveDeferredMessageBatch(receiveMode, null, sequenceNumber)
+            .next()
+            .publishOn(scheduler);
```

Comment:
I noticed this and in other places, why do you use publishOn(scheduler)? It hasn't reached user code yet, so we don't need to perform work on their scheduler in the case that it is blocking.